### PR TITLE
Support/bs4 fix comment editor

### PR DESCRIPTION
**Pull request by N1koge**
master branch
Screen Shot 2020-03-16 at 16 40 12
this branch
Screen Shot 2020-03-16 at 16 40 56

### CODE REVIEW

**Comment by yuki-takei** [comment]
image

File: src/client/styles/scss/_override-bootstrap.scss [hunk 1]
```diff
@@ -43,6 +43,9 @@ h5 {
 .nav-tabs {
   .nav-item {
     margin-right: 0.15rem;
+    a.active {
+      cursor: default
+    }
```
Comment (yuki-takei) [inline src/client/styles/scss/_override-bootstrap.scss]: これなんのためだっけ？

**Comment by N1koge** [comment]
FB対応後
master branch
Screen Shot 2020-03-19 at 15 02 35
this branch
Screen Shot 2020-03-19 at 16 09 57

File: src/client/js/components/PageComment/CommentEditor.jsx [hunk 2]
```diff
@@ -277,32 +279,37 @@ class CommentEditor extends React.Component {
...
+                      <label
+                        className="ml-2 custom-control-label"
+                        htmlFor="comment-form-is-markdown"
+                      >
+                        Markdown
+                      </label>
                     </span>
                   ) }
                 </label>
+                <span style={{ flex: 1 }} />
```
Comment (yuki-takei) [inline src/client/js/components/PageComment/CommentEditor.jsx]: `.flex-grow-1` を使う

**Pull request merged**